plugins/modules/system/xfconf.py

```diff
@@ -57,6 +57,13 @@
     default: 'no'
     aliases: ['array']
     version_added: 1.0.0
+  disable_facts:
+    description:
+    - For backward compatibility, output results are also returned as C(ansible_facts), but this behaviour is deprecated
+      and will be removed in community.general 4.0.0.
+    - This flag disables the output as facts and also disables the deprecation warning.
+    type: bool
+    default: no
```

Comment:
```suggestion
    default: no
    version_added: 2.1.0
```
I hope the sanity test for this eventually lands, as I seem to start to forget about this from time to time myself :D